app/controllers/api/articles_controller.rb

```diff
@@ -8,16 +8,22 @@ def index
 
   def show
     article = Article.find(params[:id])
-    render json: { article: article }
+    render json: { article: }
   end
 
   def create
-    new_article = Article.create(params[:article].permit!)
+    category = Category.find_by(name: params[:article][:category])
+    new_article = Article.new(article_params)
+    new_article.category = category
+    new_article.save
+    new_article.attach_image(params[:article][:image])
```

Comment:
on line 16 new method has been used why you didn't use create method

Reply:
i think we could use create method from ruby ibrary to create articles and to  save it directly to database.

Reply:
@maxarvid went through some de-bugging onthis. I understand we are defining the create method i.e. what happens when we wish to create an article. 

So we are saying it needs to find a category contain params and and image can be saved onto the record). Least thats what I think!